README.md

```diff
@@ -1,45 +1,63 @@
-machines.
+## Get started
+<!--Briefly summarize what the user will achieve in this guide.-->
+Deploy Tmate SSH server with GitHub runners.
 
+<!--Indicate software and hardware prerequisites-->
+You'll need a working [OpenStack installation](https://microstack.run/docs/single-node) to deploy
+the GitHub runners on.
 
-## Project and community
+### Set up
 
-The tmate-ssh-server Operator is a member of the Ubuntu family. It's an open source
-project that warmly welcomes community projects, contributions, suggestions,
-fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
-* [Contribute](https://charmhub.io/tmate-ssh-server/docs/contributing)
-* [Getting Started](https://charmhub.io/tmate-ssh-server/docs/getting-started)
-Thinking about using the tmate-ssh-server Operator for your next project? 
-[Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
+Follow [MicroStack's single-node](https://microstack.run/docs/single-node) starting guide to set 
+up MicroStack.
 
----
+Follow the [tutorial on GitHub runner](https://charmhub.io/github-runner) to deploy the GitHub
+runner.
 
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/tmate-ssh-server/docs).
+### Deploy
 
-## Generating src docs for every commit
+Deploy the charm.
 
-Run the following command:
+```
+juju deploy tmate-ssh-server
 
-```bash
-echo -e "tox -e src-docs\ngit add src-docs\n" >> .git/hooks/pre-commit
-chmod +x .git/hooks/pre-commit
+juju integrate tmate-ssh-server github-runner
 ```
+
+### Basic operations
+<!--Brief walkthrough of performing standard configurations or operations-->
+
+After having deployed and integrated the charm with the GitHub runner charm, the SSH connection to
+the runner VM should be accessible via [action-tmate](https://github.com/canonical/action-tmate).
+To find out more about how to configure the workflow, refer to the [Getting Started](https://github.com/canonical/action-tmate?tab=readme-ov-file#getting-started) section of action tmate.
+
+## Integrations
+<!-- Information about particularly relevant interfaces, endpoints or libraries related to the charm. For example, peer relation endpoints required by other charms for integration.--> 
+* debug-ssh: Provides GitHub runners with addresses to connect to for action-tmate.
+
+For a full list of integrations, please refer to the [Charmhub documentation](https://charmhub.io/github-runner-image-builder/integrations).
+
+## Learn more
+* [Read more](https://charmhub.io/tmate-ssh-server) <!--Link to the charm's official documentation-->
+* [Developer documentation](https://github.com/canonical/tmate-ssh-server-operator/blob/main/CONTRIBUTING.md) <!--Link to any developer documentation-->
+* [Troubleshooting](https://matrix.to/#/!DYvOMMMjuXPZRJYHdy:ubuntu.com?via=ubuntu.com&via=matrix.org)
```

Comment:
Same here, troubleshooting links to the matrix room.

Reply:
This is expected; this is what I've been telling people to link to if there's no official troubleshooting documentation/guide.

Reply:
@erinecon This links to our private team room in Matrix. Would be better to link to the Charm Development channel on Matrix_
https://matrix.to/#/#charmhub-charmdev:ubuntu.com

Reply:
Ahhhh okay. That's a good catch @varshigupta12 and @amandahla , thanks!